_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,126 @@
+
+Can your business handle the loss in revenue that may occur from slow page load speed?
+
+## Drupal 8 - Built with SEO in Mind
+
+The base of all our SEO efforts lies within the website’s architecture. There are many website engines and CMS’s to choose from and most of them will claim that they are SEO optimized. Don’t be fooled! No CMS will come search engine optimized out of the box. It may have some features, which, if configured correctly, may bring you some SEO benefits. SEO is not only about code, though it does start there. SEO is also about the continuous efforts of your marketing team. We all know that time = money. The more efficient your marketing team is in performing tasks within your CMS - the more ROI you get!
+
+A good CMS must provide means for your marketing team to work independently from your development team. Drupal 8 does just that! It provides a solid framework that can be tuned to become a powerful marketing-machine.
+
+Let’s take a look at some of the new features in core that make search engines love Drupal 8.
+
+#### Drupal 8 is Responsive out of the Box
+
+Drupal 8 comes with responsive themes in core. Now both public facing and admin facing themes are responsive and make user experience great on any device.
+
+#### Drupal 8 Page Load is Fast
+
+There has been a lot of debate about Drupal 8 vs. Drupal 7 performance / page load since the Drupal 8's release. It is a fact that vanilla Drupal 8 is running much more code than vanilla Drupal 7. It runs vendor code like Symfony, which adds some overhead. However, Drupal 8 has a significant number of performance improvements that are making up for that overhead:
+
+- **Javascript files are now loading in the footer**. Due to this change pages build up faster and user can see and use them earlier.
+
+- **[Pluggable CSS/JS aggregation and minification](https://www.drupal.org/node/2034675)** to support more optimal optimization algorithms.
+
+- **Highly improved caching**. Drupal 8 uses “cache tags” that makes caching more efficient and includes Cache Context API which provides context-based caching. This means pages load faster while ensuring that visitors always see the latest version of your site.
+
+- **BigPipe render pipeline**. Sends pages in a way that allows browsers to show them much faster. First sends the cacheable parts of the page, then the dynamic/uncacheable parts. Uses the [BigPipe technique](https://www.facebook.com/notes/facebook-engineering/bigpipe-pipelining-web-pages-for-high-performance/389414033919).
+
+These improvements have the potential to make your Drupal 8 website fly! And if after all that it is not “flying” - than you need someone to review the code that powers your website’s features. [Contact us](/contact/).
+
+#### Semantic Markup
```

Comment:
Maybe also make mention here of Drupal core's out-of-the-box support for RDF and Schema.org mappings?